### PR TITLE
[17.0][MIG] partner_address_split: Migration to 17.0

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

File: partner_address_split/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The purpose of this module is to gather generic method to spliit
```
Comment (sebalix) [inline partner_address_split/readme/DESCRIPTION.md]:
```suggestion
The purpose of this module is to gather generic method to split
```

**Review by sebalix — APPROVED** [review]
Minor comment

**Review by vvrossem — APPROVED** [review]

**Comment by yankinmax** [comment]
`pre-commit` is failing because of `partner_multi_relation`:
```
****po-python-parse-format****
partner_multi_relation/i18n/sv.po:501 Translation string couldn't be parsed correctly using str.format KeyError('kategori') - [po-python-parse-format]
```

**Comment by yankinmax** [comment]
`pre-commit` issue will be fixed here:
- [x] https://github.com/OCA/partner-contact/pull/1783

**Comment by gurneyalex** [comment]
@yankinmax Can you preserve the history of the module pre-renaming? 

https://github.com/OCA/partner-contact/pull/1543 all the commits in there are missing.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**